Branch-SDK/src/io/branch/referral/PromoViewActivity.java

```diff
@@ -0,0 +1,19 @@
+package io.branch.referral;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.webkit.WebView;
+
+/**
+ * Created by sojanpr on 2/25/16.
+ */
+public class PromoViewActivity extends Activity {
+    private WebView promoWebView_;
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        promoWebView_ = new WebView(this);
+        setContentView(promoWebView_);
+        promoWebView_.loadUrl("https://branch.io");
```

Comment:
i'm assuming this is a temp class - do you want a getter or setter for `promoWebView_`? Is this going to be a dynamic URL? or will this be used later elsewhere?

let's un-commit if not needed?